pallets/identity/src/lib.rs

```diff
@@ -633,6 +633,24 @@ decl_module! {
             }
         }
 
+        /// Assuming this is executed by the GC voting majority, adds a new cdd claim record.
+        #[weight = 950_000_000]
+        pub fn gc_add_cdd_claim(
+            origin,
+            target: IdentityId,
+            expiry: Option<T::Moment>,
+        ) -> DispatchResult {
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_add_cdd_claim(target, Claim::CustomerDueDiligence([0; 32].into()), GC_DID, expiry)
```

Comment:
If this is supposed to be a wildcard CDD claim, perhaps it's better to use the existing function `Claim::make_cdd_wildcard` to construct it.

Reply:
I've applied your suggestion; thanks for the tip!